deploy/playbooks/roles/common/templates/prod.py.j2

```diff
@@ -88,4 +89,22 @@ repos = {
     '__force_dict__': True,
 }
 
+
+# Use this to define how distributions files will be created per project
+distributions = {
+    "defaults": {
+        "DebIndices": "Packages Release . .gz .bz2",
+        "DscIndices": "Sources Release .gz .bz2",
+        "Contents": ".gz .bz2",
+        "Origin": "RedHat",
+        "Description": "",
+        "Architectures": "amd64 armhf i386 source",
+        "Suite": "stable",
+        "Components": "main",
+    },
+    "ceph": {
```

Comment:
we are missing radosgw-agent, ceph-deploy, and ceph-release here, not a deal-breaker, but maybe create a card on trello to make sure we have those ready.